src/core/midi/MidiClient.cpp

```diff
@@ -266,22 +262,21 @@ void MidiClientRaw::processParsedEvent()
 
 
 
-void MidiClientRaw::processOutEvent( const MidiEvent& event, const MidiTime & , const MidiPort* port )
+void MidiClientRaw::processOutEvent(const MidiEvent& event, const MidiTime &, const MidiPort* port)
 {
 	// TODO: also evaluate _time and queue event if necessary
-	switch( event.type() )
+	switch (event.type())
 	{
 		case MidiNoteOn:
 		case MidiNoteOff:
 		case MidiKeyPressure:
-			sendByte( event.type() | event.channel() );
-			sendByte( event.key() + KeysPerOctave );
-			sendByte( event.velocity() );
+			sendByte(event.type() | event.channel());
+			sendByte(event.key());
+			sendByte(event.velocity());
 			break;
 
 		default:
-			qWarning( "MidiClientRaw: unhandled MIDI-event %d\n",
-							(int) event.type() );
+			qWarning("MidiClientRaw: unhandled MIDI-event %d\n", (int)event.type());
```

Comment:
Should we use `static_cast` instead?

Reply:
Yes, possibly, but I tend to not touch old code too much, except for formatting tweaks when touching stuff around it.